arbitrum-docs/partials/_troubleshooting-building-partial.md

```diff
@@ -147,7 +147,18 @@ Once upon a time, Arbitrum developers were required to download supplemental pac
 
 
 
-### Is there any testnet or devnet available on Arbitrum? {#is-there-any-testnet-or-devnet-available-on-arbitrum}
+### What is a testnet or a devnet? {#what-is-a-testnet-or-a-devnet}
+<p>Testnets (or devnets) primarily serve developers who want to test out the applications they're building without having to use any real mainnet funds.</p>
+
+<p>Arbitrum testnets have the same full feature-set as the mainnet networks. They are also "true" L2s that run on top of public testnet Layer 1s, using their underlying networks for security and settlement.</p>
```

Comment:
Should we use the word testnet instead? Using it as plural may imply that we have more than one testnet available.
I'd just the following, please feel free to edit as you wish:

Nitro Goerli Rollup testnet has the same full feature-set as the mainnet networks. It is also a "true" L2 that runs on top of the Goerli testnet, using it for security and settlement